test/integration/app/prompted_nobuild.js

```diff
@@ -312,20 +313,68 @@ describe('Prompt and no build integration tests for app generator', function ()
         "/persons": {
           "get": {
             "description": "Gets `Person` objects.",
+            "responses": {
+              "200": {
+                "description": "pet response",
+                "schema": {
+                  "$ref": "#/definitions/Person"
+                }
+              }
+            }
```

Comment:
That all makes sense. I'm not too familiar with this particular test, but I do know that the swagger json [here](https://github.com/IBM-Swift/generator-swiftserver/blob/master/test/integration/app/prompted_nobuild.js#L304) is not valid.